src/inference/src/dev/core_impl.cpp

```diff
@@ -1459,6 +1459,9 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::load_model_from_cache(
                         update_config[ov::weights_path.name()] = weights_path;
                     }
                 }
+                if (model_buffer) {
+                    update_config[ov::internal::cached_model_buffer.name()] = model_buffer;
```

Comment:
should we populate `update_config` only iff plugin support import with mmap?

Reply:
I think no. Plugins don't expect they will get one more extra element in config. So we have to update all plugins in such case.